src/demo/demo.test.js

```diff
@@ -34,30 +33,33 @@ describe('Mount Demo App', () => {
     rootNode.setAttribute('id', 'demo-app')
     window.domNode = rootNode
     document.body.appendChild(rootNode)
-
-    Demo = require('./demo.js')
   })
 
   describe('by mocking Onfido SDK', () => {
-    // Mock window.Onfido
-    global.Onfido = {
-      init: () => {},
-    }
+    beforeEach(() => {
+      // Mock window.Onfido
+      window.Onfido = {
+        init: jest.fn(),
+      }
+
+      Demo = require('./demo').Demo
+    })
 
     it('mounts the Onfido Demo without crashing', () => {
-      // the component needs to be assigned to a lowercase variable to work!
-      const sdk = <Demo />
-      const sdkDemo = mount(<sdk />)
-      expect(sdkDemo.exists()).to.equal(true)
+      const sdkDemo = mount(<Demo />)
+      expect(sdkDemo.exists()).toBeTruthy()
+      expect(window.Onfido.init).toHaveBeenCalled()
```

Comment:
I might've missed something, I see expect is still used here & in line 62, but why don't we need to import `expect` from chai now?

Reply:
Yeah because we're using `jest`, which provides a builtin `expect` and different style of assertion (`toBeSomething()` instead of `to.be.something()`